### PR TITLE
Move Vixie over to netstandard2.0

### PR DESCRIPTION
**Pull request by Beyley**
This also adds a framework 4.6.1 build target for the test suite

### CODE REVIEW

File: Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ public class TextureGL : Texture, IDisposable {
         /// </summary>
         public bool IsFramebufferTexture {
             get;
-            internal init;
+            internal set;
```
Comment (Eeveelution) [inline Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs:86]: any reason for this change here?

File: Furball.Vixie/Game.cs [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+using System.Runtime.InteropServices;
 using Furball.Vixie.Backends.Shared.Backends;
 using Furball.Vixie.Helpers;
 using Kettu;
+using NativeLibraryLoader;
 using Silk.NET.Input;
 using Silk.NET.Maths;
 using Silk.NET.Windowing;
+using LibraryLoader=NativeLibraryLoader.LibraryLoader;
+using PathResolver=NativeLibraryLoader.PathResolver;
```
Comment (Eeveelution) [inline Furball.Vixie/Game.cs]: are those used anywhere? last time i checked they werent used

**Review by Eeveelution — CHANGES_REQUESTED** [review]

File: Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs (hunk 1 above)
Comment (Beyley) [reply Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs:86]: init doesnt exist on netstandard, it complains

File: Furball.Vixie/Game.cs (hunk 2 above)
Comment (Beyley) [reply Furball.Vixie/Game.cs]: Hm they must have been added when i was testing netbsd, ill remove them

File: Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs (hunk 1 above)
Comment (Eeveelution) [reply Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs:86]: right i forgot

**Review by Eeveelution — APPROVED** [review]

**Pull request merged**